### DISCUSSION TITLE
SQLite-based databases on the postgres protocol? Yes we can!

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1687] [Q&A]
@ganigeorgiev This weird.

* https://news.ycombinator.com/item?id=34517474
* https://blog.chiselstrike.com/sqlite-based-databases-on-the-postgres-protocol-yes-we-can-358e61171d65

**Comment by ganigeorgiev** [comment]
This looks interesting (it reminds me a little of [postlite](https://github.com/benbjohnson/postlite)), but the benefits for PocketBase are very small and there are no plans to switch to a fork since the cgo sqlite and pure go port for now are good enough for most of the cases. And they will become even better in the near future with the recent sqlite announcement for a new experimental backend that will allow concurrent writers - https://sqlite.org/forum/forumpost/d9b3605d7ff40cf4 (at least the [tests](https://sqlite.org/hctree/doc/hctree/doc/hctree/threadtest.wiki) are very promising).
